data/docs/openvino-ovms-serving.yaml

```diff
@@ -0,0 +1,14 @@
+metadata:
+  name: openvino-ovms-serving
+  type: how-to
+spec:
+  displayName: How do I serve a model using OpenVINO Model Server
+  appName: openvino
+  description: |-
+    Once you have created an intermediate representation using the OpenVINO model optimizer, you
+    may want to deploy a serving API in your Red Hat OpenShift Data Science environment.
+   
+    This How To highlights how this is made possible through custom resources from the OpenVINO operator
```

Comment:
Please try to limit to 120 characters. Long descriptions cause long tiles:

![image](https://user-images.githubusercontent.com/11633780/112460642-82130600-8d35-11eb-909b-b91ad725c7e3.png)

Reply:
Alright, fixed.